modules/ort/src/main/java/org/eclipse/sw360/antenna/ort/resolver/OrtResultArtifactResolver.java

```diff
@@ -98,6 +98,9 @@ private static ArtifactCoordinates mapSimpleCoordinates(String name, String vers
     }
 
     private static Optional<ArtifactSourceUrl> mapSourceUrl(Package pkg) {
+        // Prefer a VCS URL for the source URL as that usually is the pristine source from which also source artifacts
+        // are created, and only if empty fall back to a source artifact URL (which might have been processed e.g. by
+        // stripping / minimizing license headers).
         final Stream<String> urls = Stream.of(pkg.getVcsProcessed().getUrl(), pkg.getSourceArtifact().getUrl());
```

Comment:
Is the vcs url an url to a `.tar.gz` or to the github page? The second can not automatically be processed by our code and maybe even result in a html page added as source file.

Reply:
It's neither. In ORT the VCS URL is the VCS-specific URL to clone the project using a VCS tool.

Reply:
Our source URL should be a HTTP URL to a downloadable artifact. We have a `SourceUrlResolver` which downloads the URL if no sources are already present.

For a VCS URL we should add a new type of Fact.

Reply:
Good to know! Does that mean in turn that is does not make sense to consider ORT's source artifact URL as part of `mapFilename()` as it's already considered here?